crates/ruff/src/checkers/ast/mod.rs

```diff
@@ -2672,43 +2751,6 @@ where
                     }
                 }
 
-                // pyupgrade
```

Comment:
(Only applying this change to the `pyupgrade` and `flake8-django` `ast::ExprCall` rules.)